### PR TITLE
New feature to calculate temperature from Fahrenheit to Celsius

### PR DESCRIPTION
**Pull request by jdamerow**
This adds a new function to the case base that calculates the temperature in Celsius to Fahrenheit.

### CODE REVIEW

File: temps.py [hunk 1]
```diff
@@ -24,3 +24,10 @@ def convert_from_fahrenheit(temps):
         results.append((temp-32)/1.8)
 
     return results
+
+def convert_to_fahrenheit(temps):
```
Comment (jdamerow) [inline temps.py:28]: This should be named convert_from_celsius.

File: temps.py [hunk 2]
```diff
@@ -24,3 +24,10 @@ def convert_from_fahrenheit(temps):
...
+    results = []
+    for temp in temps:
+        results.append(temp*1.8+32)
```
Comment (jdamerow) [inline temps.py:31]: Here is a bug

**Review by jdamerow — COMMENTED** [review]
There a few things to change.

File: temps.py (hunk 2 above)
Comment (jophals) [reply temps.py:31]: So many bugs in this code... please fix immediately.

**Comment by jdamerow** [comment]
Ok, I fixed.

**Review by jophals — APPROVED** [review]
Great job!

**Pull request merged**